br/backup-and-restore-overview.md

```diff
@@ -115,7 +115,7 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
 |  ----  | ----  | ----- |
```

Comment:
Which version do these changes apply to?

Reply:
I think always been like this

Reply:
OK. Then we should cherry-pick it to at least the latest LTS.